src/caselawclient/identifier_resolution.py

```diff
@@ -0,0 +1,31 @@
+import json
```

Comment:
I'm a bit conflicted about where this file lives -- I think this is better than being in `models.identifiers`. Maybe it should be in `models.identifier_resolution` ?

Reply:
I agree this isn't a data model - current location seems fine to me.